### ISSUE TITLE
Cmake error in build process

### ISSUE THREAD

**Issue by carl-bb** [wxMaxima-developers/wxmaxima#1395]
Hi. I have problems to build wxmaxima 20.07.0 ( version 20.06.6 works..) .

After running `cmake -DCMAKE_INSTALL_PREFIX:PATH=/usr/local/wxmaxima-test/  ..` i have the following error:

```
[ 59%] Building CXX object src/CMakeFiles/wxmaxima.dir/ImgCell.cpp.o
/home/carl/source/wxmaxima-Version-20.07.0/src/ImgCell.cpp: In member function ‘virtual wxString ImgCell::ToRTF() const’:
/home/carl/source/wxmaxima-Version-20.07.0/src/ImgCell.cpp:287:12: error: cannot convert ‘wxStringBuffer’ {aka ‘wxStringTypeBuffer<wchar_t>’} to ‘wxStringCharType*’ {aka ‘char*’}
  287 |            wxStringBuffer(hexString, 2*imgdata.GetDataLen()));
      |            ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      |            |
      |            wxStringBuffer {aka wxStringTypeBuffer<wchar_t>}
/home/carl/source/wxmaxima-Version-20.07.0/src/ImgCell.cpp:231:67: note:   initializing argument 3 of ‘void writeHex(void*, size_t, wxStringCharType*)’
  231 | static void writeHex(void *data, size_t length, wxStringCharType *out)
      |                                                 ~~~~~~~~~~~~~~~~~~^~~
gmake[2]: *** [src/CMakeFiles/wxmaxima.dir/build.make:589: src/CMakeFiles/wxmaxima.dir/ImgCell.cpp.o] Error 1
gmake[1]: *** [CMakeFiles/Makefile2:2520: src/CMakeFiles/wxmaxima.dir/all] Error 2
gmake: *** [Makefile:182: all] Error 2
```

I'm using cmake version 3.18.2 on clearlinux os.

**Comment by KubaO** [comment]
Please indicate where does you copy of wxWindows come from: either the exact package you're using, or the full cmake command line used if you compiled it yourself.

**Comment by KubaO** [comment]
Please investigate how easy would it be to have Clearlinux as a platform on either Travis or AppVeyor. If that's relatively easy to do, then we'll add Clearlinux as a supported platform. Otherwise, it's not a supported platform, since it's not part of our CI build. In essence, it's supported if we can easily build it on CI. If it's not built there, there's no hope of long-term support. I can maybe work around this particular issue, but the build may still break at any moment - so fixing this one issue is a waste of time, since you'll come right back whenever it recurs, and lots of time will be wasted on it.

I urge you to look into how to get a Clearlinux shell under either Travis or AppVeyor, and then we can add it as a supported platform very easily, and having the build and the test suite pass will be a normal part of the process.

I suggest you leave this particular issue open for now, then create a new one for Clearlinux support **but only when you can describe** what magic is needed to get a Clearlinux setup for CI builds on either AppVeyor or Travis, since those are the CI platforms we currently use. Feel free to ask Clealinux developers what they might suggest. I have 0 time to investigate what Clearlinux is or how to get CI builds on it.

**Comment by KubaO** [comment]
Should be fixed in ae56eac3.
